packages/sdk/chainDeploy/mainnets/moonbeam.ts

```diff
@@ -38,6 +38,10 @@ export const deployConfig: ChainDeployConfig = {
         token: assets.find((a: SupportedAsset) => a.symbol === assetSymbols.STELLA)!.underlying,
         baseToken: assets.find((a: SupportedAsset) => a.symbol === assetSymbols.WGLMR)!.underlying,
       },
+      {
+        token: assets.find((a: SupportedAsset) => a.symbol === assetSymbols.CELR)!.underlying,
```

Comment:
Also here, assets need to be defined in https://github.com/Midas-Protocol/monorepo/blob/development/packages/chains/src/moonbeam/assets.ts